httpfuzzerprocessor/pitchWork.py

```diff
@@ -0,0 +1,84 @@
+# Version 1.0
+# @author RUFFENACH Timothée
+# Script inspired from https://02108124551050482571.googlegroups.com/attach/54c6e34f6fe20/message_processor.js?part=0.1&view=1&vt=ANaJVrEJuACewYorhYYa_zyhyMSug06pmlERCqfYdLsukQBC3OW3LATuXG1WHk_Fw9a0nhexG8ykFDuFgBGYrKAg_pOQ61M36MwC9SOBGvK4KLZn3eDkNzY (dot run on owasp 2.12.0)
```

Comment:
> dot run on owasp 2.12.0

?

Reply:
legacy script in javascript don't run with owasp 2.12.0

Reply:
That's not a ZAP issue that's a JRE issue. If you're using a Java 11 JRE it'll still run.

Also OWASP is the organization ZAP is the project/product :wink:

Reply:
Link to the thread/post rather than the attachment, which should be more reliable.

Reply:
I don't found the original post.